core/src/main/java/io/github/tcdl/msb/support/Utils.java

```diff
@@ -56,8 +58,8 @@ public static String toJson(Object object, ObjectMapper objectMapper) {
         try {
             return objectMapper.writeValueAsString(object);
         } catch (JsonProcessingException e) {
-            LOG.error("Failed parse to JSON from: " + object, e);
-            throw new JsonConversionException(e.getMessage());
+            LOG.error("Failed to parse to JSON object: [{}] ", object);
```

Comment:
I see your point and it's a valid one. Though it's somewhat hard to relate those messages. Maybe we could twist in message id in some way to be available for responder server? Like: http://skipy.ru/useful/logging.html#log4j_ndc